modules/packit-api.nix

```
@@ -109,6 +115,19 @@ let
         };
       };
 
+      runner = rec {
```

```suggestion
      runner = {
```

Doesn't seem necessary?

---

done